### PR TITLE
Larger fixes for sc errors

### PR DESCRIPTION
**Pull request by ndjc**
For editors of specs, remember to run ShapeChecker on your vocabularies and shapes. The attached diffs suggest some possible changes to various specs to reduce ShapeChecker errors.

### CODE REVIEW

**Comment by berezovskyi** [comment]
I will review once you mark the draft ready for review, @ndjc

File: specs/rm/requirements-management-shapes.ttl [hunk 1]
```diff
@@ -172,28 +170,17 @@ or to satisfy a contract, standard, specification, or other formally imposed doc
...
 
 <#shortTitle> a oslc:Property ;
     oslc:name "shortTitle" ;
     oslc:occurs oslc:Zero-or-one;
-    dcterms:description "Short name identifying a resource, often used as an abbreviated identifier for presentation to end-users. SHOULD include only content that is valid inside an XHTML &lt;span&gt element."^^rdf:XMLLiteral ;
+    dcterms:description "Short name identifying a resource, often used as an abbreviated identifier for presentation to end-users. SHOULD include only content that is valid inside an XHTML &lt;span&gt; element."^^rdf:XMLLiteral ;
     oslc:propertyDefinition  oslc:shortTitle ;
     oslc:valueType rdf:XMLLiteral .
 
-<#modifiedBy> a oslc:Property ;
```
Comment (berezovskyi) [inline specs/rm/requirements-management-shapes.ttl]: was it used anywhere? fine by me if not, this is not an RDF property.

File: specs/rm/requirements-management-vocab.ttl [hunk 2]
```diff
@@ -17,7 +17,7 @@ oslc_rm:
   a owl:Ontology ;
   dcterms:title "The OSLC Requirements Management(RM) Vocabulary" ;
   dcterms:description "All vocabulary URIs defined in the OSLC Requirements Management (RM) namespace."^^rdf:XMLLiteral ;
-  dcterms:source <https://github.com/oslc-op/oslc-specs/blob/master/specs/blob/master/rm/requirements-management-vocab.ttl> ;
+  dcterms:source <https://raw.githack.com/oslc-op/oslc-specs/master/specs/rm/requirements-management-vocab.ttl> ;
```
Comment (berezovskyi) [inline specs/rm/requirements-management-vocab.ttl]: I think it should be https://github.com/oslc-op/oslc-specs/raw/master/specs/rm/requirements-management-vocab.ttl instead.

File: specs/rm/requirements-management-shapes.ttl [hunk 3]
```diff
@@ -144,8 +142,8 @@ or to satisfy a contract, standard, specification, or other formally imposed doc
 <#title> a oslc:Property ;
     oslc:name "title" ;
     oslc:propertyDefinition dcterms:title   ;
-    oslc:occurs oslc:exactly-one;
-    dcterms:description "Title (reference: Dublin Core) of the resource represented as rich text in XHTML content. SHOULD include only content that is valid inside an XHTML &lt;span&gt element. "^^rdf:XMLLiteral ;
+    oslc:occurs oslc:Exactly-one;
```
[the remark this answers is not in the document]
Comment (ndjc) [reply specs/rm/requirements-management-shapes.ttl]: Yes, it was.

File: specs/rm/requirements-management-shapes.ttl (hunk 1 above)
Comment (ndjc) [reply specs/rm/requirements-management-shapes.ttl]: No, it was not used.

File: specs/rm/requirements-management-vocab.ttl (hunk 2 above)
Comment (ndjc) [reply specs/rm/requirements-management-vocab.ttl]: Agreed.

**Comment by ndjc** [comment]
I also see your CI test is failing due to the error trying to read the foaf vocabulary document. I added a work-around for that - is the CI build not using the latest ShapeChecker code?

**Comment by ndjc** [comment]
And the CI build is always running ShapeChecker on Core - for a pull request on one of the other domains, we should run ShapeChecker on the vocabulary and shapes for that domain.

**Comment by berezovskyi** [comment]
Wonderful, thanks! We should strive to extend the script to validate all RDF sources in the repo.

**Comment by berezovskyi** [comment]
@ndjc I think I got what you meant.

1) If you feel that your branch is behind master, you need to merge master into your branch:

```
git checkout master
git pull
git checkout larger_fixes_for_SC_errors
git merge master
```

This is also the right process to use if the green button on the PR turns grey and Github tells you that there is a merge conflict for you to resolve by hand.

2) The idea is that master shall always be "green". Therefore, you should always fix your PR till it's green and thus it's normal to run SC on everything. So, no need to make the build process for PRs any different than master. We are in limbo till the master is green, though.

**Comment by berezovskyi** [comment]
I took the liberty of doing step 1 for you and pushing the merge commit. Please squash and merge once you are ready.

**Pull request merged**